Firmware/Marlin_main.cpp

```diff
@@ -9792,7 +9792,7 @@ static void wait_for_heater(long codenum, uint8_t extruder) {
 	while ((!cancel_heatup) && ((residencyStart == -1) ||
 		(residencyStart >= 0 && (((unsigned int)(_millis() - residencyStart)) < (TEMP_RESIDENCY_TIME * 1000UL))))) {
 #else
-	while (target_direction ? (isHeatingHotend(tmp_extruder)) : (isCoolingHotend(tmp_extruder) && (CooldownNoWait == false))) {
```

Comment:
I wouldn't remove CooldownNowait.
See https://github.com/prusa3d/Prusa-Firmware/pull/2060/files#diff-a2e2d6e9dab1e37ea9a2882433e2b8f70af751b3c3445d5e997d11a93d4b28b3R10135 and the comments in the PR.

Reply:
But its not defined in this scope 🤔the build will fail

Reply:
Hmm I’m pretty sure I saw the build fail due to the variable being undefined. I can double check tomorrow.

Reply:
Yeah, I'm sure it was broken. I just remembered that we had a PR fixing this by actually using/fixing CooldownNoWait, which is IMHO better.

Reply:
Fixed. The PR now only resolves the undefined variable `tmp_extruder`.